girder_worker/app.py

```diff
@@ -31,8 +31,8 @@
     _update_status,
     is_builtin_celery_task,
     is_revoked
-
 )
+from girder_worker_utils.transform import ResultTransform
```

Comment:
Maybe this is ridiculous, but I'm a little concerned about introducing this dependency on girder_worker_utils here - i'd have to think a little harder to make a cogent argument for why.  Would it be a problem to check if there is a `job` attr on the girder_result_hook object that is set to `None`  rather than directly checking the instance type?

Reply:
If you think that's a better architecture, I will change it, I leave that decision to you.

Since girder_worker already depends on girder_worker_utils, what's the issue with referencing it here?

Reply:
No you're right,  its probably fine as is